app/models/keystore.rb

```diff
@@ -91,4 +92,9 @@ def self.decrement_value_for(key, amount = -1)
   def self.decremented_value_for(key, amount = -1)
     self.incremented_value_for(key, amount)
   end
+
+  def self.validate_input_key(key)
+    exception = ActiveRecord::ValueTooLong.new('50 characters is the maximum allowed for key')
```

Comment:
I don't have a better solution, but it concerns me that now this length is validated here and on line 4. They could get out of sync. (Also, it's weird to assign an exception to a variable before raising it, I've never seen that before. Was it just for line length?)

Reply:
Yeap, I've looked through the app and how this class is using and found that it is using as service module, without existing any class instances so adding validation on line 4 for class instances is useless, but that stub database_consistency gem issue. By the way, that issue is right, as we have a limit in db for this field, so I decided to add this exception as we inserting data here through SQL without AR. If this gets out of sync we may remove it, no matter)

Reply:
Maybe just move the `50` into a constant the class. Then it's not defined multiple places that can get out of sync, and anyone touching one place can't help but notice it's a shared value and that there's other uses to look at.

Reply:
Resolved